docs/typescript/testing.md

```diff
@@ -351,3 +354,12 @@ await worker.runUntil(
   testEnv.workflowClient.execute(functionToTest, workflowOptions) // Throws WorkflowFailedError
 );
 ```
+
+### Testing with Jest
+
+Although these testing docs use [Mocha](https://mochajs.org/), you can also use [Jest](https://jestjs.io/).
+There are a couple of caveats for testing with Jest:
+
+1. The Temporal TypeScript SDK only supports Jest `>= 27.0.0`.
+2. You need to run Jest with the `NODE_OPTIONS` environment variable set to `--experimental-vm-modules`, for example: `NODE_OPTIONS=--experimental-vm-modules jest`.
```

Comment:
This won't be relevant in the latest release AFAICT

Reply:
Removed, and edited PR title to match current state